python/cudaq/kernel/ast_bridge.py

```diff
@@ -42,6 +42,12 @@ def pushScope(self):
     def popScope(self):
         self.symbolTable.pop()
 
+    def add(self, symbol, value, level=-1):
```

Comment:
Suggest changing function name to be more descriptive, something like `addSymbol` or `addToSymbolTable`...